index.js

```diff
@@ -4,7 +4,7 @@
 const INDENT_REGEX = /^(?:( )+|\t+)/;
 
 function getMostUsed(indents) {
-	let result = 0;
+	let result = null;
```

Comment:
I prefer just using `undefined`.
```suggestion
	let result;
```